packages/node/src/core/evm/wallet.ts

```diff
@@ -20,6 +20,13 @@ function getPathFromIndex(index: number | string) {
   return `m/0/0/${index}`;
 }
 
+export function computeProviderId(provider: ethers.providers.JsonRpcProvider) {
+  const masterHdNode = ethers.utils.HDNode.fromMnemonic(security.masterKeyMnemonic);
+  const designatorHdNode = masterHdNode.derivePath('m');
```

Comment:
This line probably does nothing. You can remove it unless you wanted to be explicit.